generator/_generate.py

```diff
@@ -72,7 +72,7 @@ def run(ns):
             name, version = facade['Name'], facade['Version']
         except (KeyError, TypeError, ValueError) as err:
             raise AppError('cannot retrieve name and version for facade')
-        if name in _BLACKLIST:
+        if name in _BLACKLIST or facade['Methods'] is None:
```

Comment:
curious why this is the case...can you add a code comment plz

Reply:
Done.